controllers/subscriber/subscriberControllers.js

```diff
@@ -111,7 +118,11 @@ export class EventSubscriberController extends BaseSubscriber {
 
 	hostConnected() {
 		// delay subscription otherwise import/upgrade order can cause selection mixin to miss event
-		requestAnimationFrame(() => this._subscribe());
+		this._updateComplete = promiseWithResolve();
```

Comment:
What's the rationale for `promiseWithResolve`? Can we just do this?
```javascript
this._updateComplete = new Promise(resolve => {
    this._subscribe();
    resolve();
});
```

Reply:
Umm... yes 😅. Not sure why I thought that wasn't an option earlier, thanks!